### ISSUE TITLE
Make pullquote clickable?

### ISSUE THREAD

**Issue by jeffam** [a-fro/drupal_pullquotes#2]
You know what would be cool? If users could click the pullquote and it would scroll to and/or highlight the original text.

That would add some usability (giving context) to the already nice aesthetics of the pullquotes.

**Comment by a-fro** [comment]
Yes, I also think that there should be a way to assign an attribution to a quote, within a pullquote. My primary concern at the moment would be accessibility for adding any interface changes. I'm not sure if there's a standard practice for giving context to a pullquote.
